### DISCUSSION TITLE
can't delete organization

### DISCUSSION THREAD

**Discussion by extremeleadprogram** [postalserver/postal#3260] [Help with using Postal] [answered]
Postal 3.3.4 on Ubuntu 22.04.

I'm trying to delete an organization but it just sits there like this forever:

![image](https://github.com/user-attachments/assets/f955317d-ce63-4d6f-af73-d7ab7d6fc508)

I'm logged in as an admin.

Any ideas on things to check?

**Comment by willpower232** [comment]
You will either need to see the logs in the terminal with `postal logs web` or open the browser developer tools and see what happened in the network tab when you clicked the delete button.

**Comment by extremeleadprogram** [comment]
Here's what I see in the log:

![image](https://github.com/user-attachments/assets/8b041e28-8b77-490e-a318-119337ca32b9)

I'm guessing it's this part:  exception_message=Validation failed: Owner must exist

I did have a user issue where I deleted the original user and made a new one.

How do I change the user (owner?) of that organization?

**Reply by willpower232** [reply, marked as answer]
I don't know enough rails or ruby to know why it needs that, not sure if you can fix this in the UI.

I would recommend fixing this in the database, having a look in the main `postal` database to find the organisation and change the owner_id field.

**Reply by extremeleadprogram** [reply]
That fixed it just so anyone else knows if they run across this. I just changed the owner_id field to an actual existing user ID then I was able to delete the organization fine. Thanks for your help @willpower232 .
